src/clarity-angular/button/button-group/button-group.module.ts

```diff
@@ -5,16 +5,27 @@
  */
 
 import {CommonModule} from "@angular/common";
-import {NgModule} from "@angular/core";
+import {NgModule, Type} from "@angular/core";
 
 import {ClrIconModule} from "../../icon/icon.module";
 import {ClrCommonPopoverModule} from "../../popover/common/popover.module";
+import {ClrButtonGroup} from "./button-group";
+import {ClrButton} from "./button";
 
-import {CLR_BUTTON_GROUP_DIRECTIVES} from "./index";
+export const CLR_BUTTON_GROUP_DIRECTIVES: Type<any>[] = [ClrButton, ClrButtonGroup];
 
 @NgModule({
     imports: [CommonModule, ClrIconModule, ClrCommonPopoverModule],
     declarations: [CLR_BUTTON_GROUP_DIRECTIVES],
     exports: [CLR_BUTTON_GROUP_DIRECTIVES]
 })
 export class ClrButtonGroupModule {}
+
+/* tslint:disable variable-name */
+/** @deprecated since 0.11 */
+export const Button = ClrButton;
```

Comment:
This might seem like a nitpick, but these deprecated exports are pure API considerations, I believe they should have stayed in the index file, not the module one.

Reply:
I wanted that originally too, but then I ran into circular dependency issues and moved any declarations from the index files. It makes the index cleaner since we don't have to import and then export, which was part of my goal as well.